### PR TITLE
[IMP] Documents: improve and update the Documents documentation

### PR DESCRIPTION
**Pull request by Donapi**
[ID: 2899532](https://www.odoo.com/web#id=2899532&cids=1&model=project.task&view_type=form)

### CODE REVIEW

**Comment by xpl-odoo** [comment]
You forgot to do `make html` before pushing the PR :)
chrome_ITjPWpKIon

File: content/applications/finance/documents.rst [hunk 1]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
+   --> Actions`.
 
 Set the conditions
 ------------------
 
-After naming your workflow action, you can set the conditions that trigger the appearance of the
-action button on the right-side panel when selecting a file.
+You can :guilabel:`Create` a new :guilabel:`Action` or edit an existing one. You can define the
+:guilabel:`Action Name` and then set the conditions that trigger the appearance of the
+:guilabel:`action button` on the right-side panel when selecting a file.
```
Comment (xpl-odoo) [inline content/applications/finance/documents.rst:130]:
```suggestion
action button (:guilabel:`▶`) on the right-side panel when selecting a file.
```

File: content/applications/finance/documents.rst [hunk 2]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
 
 There are three basic types of conditions you can set:
 
-#. **Tags**: you can both use the *Contains* and *Does not contain* conditions, meaning the files
-   *must have* or *mustn't have* the tags set here.
+#. **Tags**: you can both use the :guilabel:`Contains` and :guilabel:`Does not contain` conditions,
```
Comment (xpl-odoo) [inline content/applications/finance/documents.rst:134]:
```suggestion
#. :guilabel:`Tags`: you can both use the :guilabel:`Contains` and :guilabel:`Does not contain` conditions,
```
The same applies to Contact and Owner below

File: content/applications/finance/documents.rst [hunk 3]
```diff
@@ -120,4 +208,14 @@ Select the *Actions* tab to set up your action. You can simultaneously:
...
+
+Digitize documents with AI and optical character recognition (OCR)
+==================================================================
+
+Documents available in the Finance workspace can be digitized. Select the document you want to
+digitize, click on :guilabel:`Create Bill`, :guilabel:`Create Customer Invoice` or
+:guilabel:`Create credit note`, and then click on :guilabel:`Send for Digitization`.
+
+.. seealso::
+   - `AI-powered document digitization <https://www.odoo.com/documentation/15.0/applications/finance/accounting/payables/supplier_bills/invoice_digitization.html>_
```
Comment (xpl-odoo) [inline content/applications/finance/documents.rst:221]:
```suggestion
   :doc:`AI-powered document digitization <../accounting/payables/supplier_bills/invoice_digitization>
```
[You should link to other docs this way](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks). 
However, I assume the digitization page wasn't merged yet I assume, so this link won't work either in any case.

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Nice update!
Please check that everything applies to v16 (many screenshots will have to be re-taken I'm afraid 😞)

File: content/applications/finance/documents.rst [hunk 4]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
+   on :guilabel:`Journals` and define each journal independently to allow automatic synchronization.
+
+.. image:: documents/accounting-files-centralization.png
+   :align: center
+   :alt: Enable the centralization of files attached to your accounting.
+
+Workspaces
+==========
+
+Workspaces are defined by hierarchical folders having their own set of tags and actions. By-default
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:39]: I simplified the sentence and just changed to "are hierarchical folders"

**Comment by Donapi** [comment]
The numbering was not working correctly in the "set the conditions" section (line 131). The only way to have the numbering was to not respect the 100 character per line rule...

File: content/applications/finance/documents.rst [hunk 5]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
+workspaces exist, but you can create your own by going to :menuselection:`Configuration -->
+Workspaces` and clicking on :guilabel:`CREATE`.
+
+.. note::
+   :guilabel:`Sub-workspaces` can also be created, edited or deleted, by clicking on the
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:49]: Sub-workspaces can only be created from the left menu so it looks interesting to me to mention it.

File: content/applications/finance/documents.rst [hunk 6]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
+:guilabel:`Lock` or :guilabel:`Split`. It is also possible to :guilabel:`Open Chatter` or
+:guilabel:`Archive` the document.
+
+.. image:: documents/right-panel-options.png
+   :align: center
+   :alt: right panel options
+
+The :guilabel:`Open chatter` option allows to :guilabel:`Send Message` (to everyone following the
+document), to :guilabel:`Log Note`, or to :guilabel:`Schedule activity`. You can also
+:guilabel:`Attach files`, :guilabel:`Follow/Unfollow` the document or show followers.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:82]: removed

[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:82]: from my side it has been removed...

File: content/applications/finance/documents.rst [hunk 7]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
 
 Workflow actions
 ================
 
-Workflow actions are automated actions you can create and customize at the level of each
-workspace. They appear next to a file whenever it meets the criteria you set. That way you can,
-for example, add tags to a file or move it to another workspace with a single click. Workflow
-actions help you streamline the management of your documents and your overall business operations.
+For each workspace, it is possible to configure specific workflow actions that are automated actions
+that can be created and customized at the level of each workspace. Ie: create documents, process
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:100]: Updated from my side but it looks like I didn't push the changes... not sure what happened...

File: content/applications/finance/documents.rst [hunk 8]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
+bills, sign, organize files, etc.
+
+They appear on the right panel when it meets the criteria you set. That way, you can, for example,
+add tags to a file or move it to another workspace with a single click. Workflow actions help you
+streamline the management of your documents and your overall business operations.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:105]: done

File: content/applications/finance/documents.rst [hunk 9]
```diff
@@ -9,43 +9,130 @@ various workspaces.
...
+.. image:: documents/right-panel-options.png
+   :align: center
+   :alt: right panel options
+
+The :guilabel:`Open chatter` option allows to :guilabel:`Send Message` (to everyone following the
+document), to :guilabel:`Log Note`, or to :guilabel:`Schedule activity`. You can also
+:guilabel:`Attach files`, :guilabel:`Follow/Unfollow` the document or show followers.
+
+Then, you can modify the name of your file by clicking on :guilabel:`Document`. A
+:guilabel:`Contact` or an :guilabel:`Owner` can be assigned. The related :guilabel:`Workspace` can
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/documents.rst:85]: updated. I didn't respect the max of 100 character per line as it didn't rendered ok in html...

File: content/applications/finance/documents.rst [hunk 10]
```diff
@@ -9,47 +9,119 @@ various workspaces.
 
 .. seealso::
    - `Odoo Documents: product page <https://www.odoo.com/app/documents>`_
+   - `Odoo Tutorials: Documents basics <https://www.odoo.com/slides/slide/documents-basics-674>`_
+   - `Odoo Tutorials: Using Documents with your Accounting App <https://www.odoo.com/slides/slide/using-documents-with-your-accounting-app-675?fullscreen=1#>`_
```
Comment (AntoineVDV) [inline content/applications/finance/documents.rst:13]: Lines over 100 characters are hard to maintain (read, modify, review, ...).
```suggestion
   - `Odoo Tutorials: Using Documents with your Accounting App
     <https://www.odoo.com/slides/slide/using-documents-with-your-accounting-app-675?fullscreen=1#>`_
```

File: content/applications/finance/documents.rst [hunk 11]
```diff
@@ -9,47 +9,119 @@ various workspaces.
...
+
+Configuration
+=============
+
+By going to :menuselection:`Documents --> Configuration --> Settings`, you can enable the
+centralization of files attached to a specific area of your activity. For example, by ticking
+:guilabel:`Human Resources`, your HR documents are automatically available in the HR workspace,
+while documents related to Payroll are automatically available in the Payroll sub-workspace . You
+can change the default workspace by using the dropdown menu and edit its properties by clicking the
+internal link button (:guilabel:`➔`)
```
Comment (AntoineVDV) [inline content/applications/finance/documents.rst:23]:
```suggestion
while documents related to Payroll are automatically available in the Payroll sub-workspace. You
can change the default workspace by using the dropdown menu and edit its properties by clicking the
internal link button (:guilabel:`➔`).
```

File: content/applications/finance/documents.rst [hunk 12]
```diff
@@ -9,47 +9,119 @@ various workspaces.
...
+
+Workspaces
+==========
+
+Workspaces are hierarchical folders having their own set of tags and actions. Default workspaces
+exist, but you can create your own by going to :menuselection:`Documents --> Configuration -->
+Workspaces` and clicking on :guilabel:`Create`.
+
+.. note::
+   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces`can be created, edited, or deleted by
```
Comment (AntoineVDV) [inline content/applications/finance/documents.rst:49]: I didn't check, but this markup is probably broken.
```suggestion
   :guilabel:`Workspaces` and :guilabel:`Sub-workspaces` can be created, edited, or deleted by
```

File: content/applications/finance/documents.rst [hunk 13]
```diff
@@ -9,47 +9,119 @@ various workspaces.
...
+   :alt: right panel options
+
+Then, you can modify the name of your file by clicking on :guilabel:`Document`. A
+:guilabel:`Contact` or an :guilabel:`Owner` can be assigned. The related :guilabel:`Workspace` can
+be modified and it is possible to access the related :guilabel:`Journal Entry` or to add
+:guilabel:`Tags`.
+
+.. note::
+   - The :guilabel:`Contact` is the person related to the document and assigned to it. He can only view the document and not modify it. I.e.: an existing supplier in your database is the contact for their bill.
+   - The person who creates a document is, by default :guilabel:`Owner` of it and has complete rights to the document. It is possible to replace the owner of a document. I.e.: an employee must be owner of a document to be able to see it in "My Profile".
```
Comment (AntoineVDV) [inline content/applications/finance/documents.rst:87]: These lines don't fit on a 1080p screen!

File: content/applications/finance/documents.rst [hunk 14]
```diff
@@ -9,47 +9,119 @@ various workspaces.
...
-action button on the right-side panel when selecting a file.
+You can :guilabel:`Create` a new :guilabel:`Action` or edit an existing one. You can define the
+:guilabel:`Action Name` and then set the conditions that trigger the appearance of the action button
+(:guilabel:`▶`) on the right-side panel when selecting a file.
 
 There are three basic types of conditions you can set:
 
-#. **Tags**: you can both use the *Contains* and *Does not contain* conditions, meaning the files
-   *must have* or *mustn't have* the tags set here.
+#. :guilabel:`Tags`: you can both use the :guilabel:`Contains` and :guilabel:`Does not contain` conditions, meaning the files *must have* or *mustn't have* the tags set here.
```
Comment (AntoineVDV) [inline content/applications/finance/documents.rst:120]: Too long.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: content/applications/finance/documents.rst (hunk 10 above)
Comment (Donapi) [reply content/applications/finance/documents.rst:13]: when I split in two lines, I have an error: WARNING: Inline interpreted text or phrase reference start-string without end-string. 
image

Comment (AntoineVDV) [reply content/applications/finance/documents.rst:13]: Try removing the white space before `<https...`. The continuation line should be aligned with the **first character** of the line above, not with the first *alphabetical* character.

Comment (Donapi) [reply content/applications/finance/documents.rst]: solved!

**Comment by AntoineVDV** [comment]
@xpl-odoo It would have been blocked but the r+ was just never received by GitHub, that happens. Please r+ after fixing the conflict @Donapi ;)

@robodoo delegate+

**Comment by Donapi** [comment]
@robodoo r+

**Pull request closed without merging**